### DISCUSSION TITLE
request help: how can I wrap my response body

### DISCUSSION THREAD

**Discussion by wtjperi2003** [apache/apisix#5705] [Q&A]
### Issue description

for example: response from upstream service is:  `{"name":"abc","id":124}`,  then at the apisix. I want it to wrap it: `{"success":true,data:{"name":"abc","id":124}}`  

### Environment

- apisix version (cmd: `apisix version`):
- OS (cmd: `uname -a`):
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have:
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner):
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

**Comment by tzssangglass** [comment]
refer to: https://github.com/apache/apisix/issues/4837#issuecomment-900101911
